specification/elastic/resource-manager/Microsoft.Elastic/preview/2022-05-05-preview/elastic.json

```diff
@@ -0,0 +1,2302 @@
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderDefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "AllTrafficFilters_list": {
+            "$ref": "./examples/AllTrafficFilters_list.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Elastic/monitors/{monitorName}/listAssociatedTrafficFilters": {
+      "post": {
+        "tags": [
+          "TrafficFilter"
+        ],
+        "summary": "Get the list of all associated traffic filters for the given deployment.",
+        "operationId": "listAssociatedTrafficFilters_list",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MonitorNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ElasticTrafficFilterResponse"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderDefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "listAssociatedTrafficFilters_list": {
+            "$ref": "./examples/AssociatedFiltersForDeployment_list.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Elastic/monitors/{monitorName}/createAndAssociateIPFilter": {
+      "post": {
+        "tags": [
+          "TrafficFilter"
+        ],
+        "summary": "Create and Associate IP traffic filter for the given deployment.",
+        "operationId": "createAndAssociateIPFilter_Create",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MonitorNameParameter"
+          },
+          {
+            "in": "query",
+            "name": "ips",
+            "description": "List of ips",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "name",
+            "description": "Name of the traffic filter",
```

Comment:
Move to body. action parameters must be in the body , not in the query : 

[https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/proxy-api-reference.md#resource-action-requests](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/proxy-api-reference.md#resource-action-requests)

Reply:
The same changes in the private swagger repo which were approved and merged (in [PR1](https://github.com/Azure/azure-rest-api-specs-pr/pull/7741) and [PR2](https://github.com/Azure/azure-rest-api-specs-pr/pull/7198) ). Can this be suppressed, or can we get an exception? @raosuhas

Reply:
Is there a reason you need the exception ? Why cant this be fixed ? Was this specific issue raised in that PR ? Also is that change already released to external customers ?

Reply:
The change is already in production and this specific issue was not raised in the previous PR. Doing this will be a breaking change now.